htdocs/website/index.php

```diff
@@ -1594,7 +1594,7 @@
 			}
 
 
-			$dataposted = trim(GETPOST('WEBSITE_HTML_HEADER', 'restricthtmlallowunvalid'));
+			$dataposted = trim(GETPOST('WEBSITE_HTML_HEADER', 'none'));		// Must accept tags like '<script>' and '<link>'
```

Comment:
Because the none is not clear of what we can sanitize or not, i have introduced a new possible parameter 'restricthtmlallowlinkscript' ?
Is it ok with it indtead of none on your side ?

Reply:
Yes, it works well. I sent a new PR to apply the 'restrictHtmlAllowLinkScript' parameter.